src/main/java/science/icebreaker/config/SecurityConfig.java

```diff
@@ -0,0 +1,81 @@
+package science.icebreaker.config;
```

Comment:
Missing documentation! Is this a standard config file (where we can find docs online -> provide link) and/or what settings did you change.